charts/all-in-one/templates/rudolf-service.yaml

```diff
@@ -75,6 +98,9 @@ spec:
       app: 9c-rudolf
   securityGroups:
     groupIds:
-    - {{ $.Values.rudolfService.securityGroupId }}
+{{ if not .Values.rudolfService.db.local }}
+    - {{ $.Values.rudolfService.db.securityGroupId }}
 {{ end }}
+    - {{ $.Values.rudolfService.securityGroupIds.podToService }}
+    {{- toYaml $.Values.rudolfService.securityGroupIds.extra | nindent 4 }}
 {{ end }}
```

Comment:
have you ever tried without any security groups? i dont think it's okay for empty list

Reply:
Well.. I have never tried without any security groups. (And I don't it is not recommended way at least database). I'm working on making service and service security group optional.

Reply:
got it, i will fix it someday when it goes wrong. i think if we use database as a pod for a test network, we dont need sg

Reply:
I applied it.

> i think if we use database as a pod for a test network, we dont need sg

The DB sg is already optional!

```
{{ if not .Values.rudolfService.db.local }}
    - {{ $.Values.rudolfService.db.securityGroupId }}
{{ end }}
```